packages/trpc/server/routers/loggedInViewer/integrations.handler.ts

```diff
@@ -114,7 +114,7 @@ export const integrationsHandler = async ({ ctx, input }: IntegrationsOptions) =
     }
   }
 
-  const enabledApps = await getEnabledApps(credentials);
+  const enabledApps = await getEnabledApps(credentials, onlyInstalled);
```

Comment:
If the `onlyInstalled` parameter is passed then also only grab the metadata of installed apps.